.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -335,28 +328,23 @@ jobs:
             cargo_dir: C:\Users\runneradmin\.cargo
             cc: cl
             cxx: cl
-            # The D drive runs out of space when building dependencies with vcpkg.
             workspace: C:\cxx-qt
 
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.name }}
     env:
-        # Match the deployment target that Qt was built with via vcpkg, otherwise the following error occurs
         # ld: warning: object file (LIB) was built for newer macOS version (12.7) than being linked (12.0)
-        MACOSX_DEPLOYMENT_TARGET: 12.7
+        MACOSX_DEPLOYMENT_TARGET: ${{ matrix.macosx_deployment_target }}
         # sccache is around 180-300M in size for a normal build
         # With GitHub caches we have a 10 GB limit / 6 conditions = 1666 MB
         # Allow a larger cache size so that code in branches can be cached
         # but still leave room for the tools cache
         SCCACHE_CACHE_SIZE: 600M
 
     steps:
-    # On Windows, the D drive that the workspace is on by default runs out of space when
-    # vcpkg builds Qt, so the repo needs to be cloned to the C drive. GitHub's checkout
-    # action does not support cloning to a path outside GITHUB_WORKSPACE:
     # https://github.com/actions/checkout/issues/197
```

Comment:
oops also remove this line linking to the github issue as it's not relevant anymore